ecs-cli/modules/cli/local/down_app.go

```diff
@@ -81,8 +81,7 @@ func downComposeLocalContainers() error {
 }
 
 func downLocalContainersWithFilters(args filters.Args) error {
-	ctx, cancel := context.WithTimeout(context.Background(), docker.TimeoutInS)
-	defer cancel()
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Can you change this back to a `context.WithTimeout`? We want the `ContainerList` operation to also have a 30 second deadline.